reference/interbit-cli/start.adoc

```diff
@@ -0,0 +1,52 @@
+= `start`
+
+A command-line command that starts the Developer mode hypervisor.
+
+
+== Syntax
+
+[listing,subs="quotes"]
+interbit **start** [--config _path/to/interbit.config.js_] \
+               [--port _number_] \
+               --dev \
+               --no-watch
+
+
+== Options
+
+[horizontal]
+[.api.p]`--config` [.api.t]__String__ [.api.o]**Optional** [.api.d]**Default=./interbit.config.js**::
+The path to an Interbit configuration file to use for deployment.
+
+[.api.p]`--port` [.api.t]__Integer__ [.api.o]**Optional** [.api.d]**Default=5000**::
+The port that your Interbit app uses to communicate.
```

Comment:
~app~ node

Reply:
We don't refer to it often enough to have a standard. If we say `interbit/interbit` to mean the repo, we should say repo and provide its full URL. If something else is meant, we should describe that.